FAULT TYPE: OPENSTACK_THROW_EXCEPTION-INSTANCE
TARGET COMPONENT: /nova/compute/api.py
TARGET CLASS: API
TARGET FUNCTION DEF: _create_instance+1187;12
FAULT POINT: self.compute_task_api.schedule_and_build_instances(context, build_requests=build_requests, request_spec=request_specs, image=boot_meta, admin_password=admin_password, injected_files=injected_files, requested_networks=requested_networks, block_device_mapping=block_device_mapping, tags=tags)
